examples/experimental/terrain/README.md

```diff
@@ -0,0 +1,30 @@
+This is a minimal standalone version of the TerrainLayer example
```

Comment:
Since this is in loaders.gl, the example should be about the `TerrainLoader`, not the `TerrainLayer`.
```suggestion
This an example of using the `TerrainLoader`
```